Fix RTL screen support

medyas opened the pull request:
upgraded SVG package and fixed the support for RTL screens

xsahil03x commented:
@medyas hey thanks for your contribution but can you please fix the build before I merge it.

medyas commented:
There was a bug with the indicators in RTL screen. i fixed it, and updated the example main file with localization string and as the `easy_localization` package use the `shared_pref` package and they only work on OSs, it caused the test to fail(sorry for that, i did not check). I removed the localization example file and packages.

xsahil03x commented:
Thanks again :+1:
